lib/core/text/flat.nit

```diff
@@ -1308,37 +1308,44 @@ private class FlatBufferCharIterator
 end
 
 redef class CString
-	redef fun to_s
-	do
-		return to_s_with_length(cstring_length)
-	end
 
-	redef fun to_s_with_length(length)
-	do
-		assert length >= 0
-		return clean_utf8(length)
-	end
+	# Get a `String` from the data at `self` copied into Nit memory
+	#
+	# Require: `self` is a null-terminated string.
+	redef fun to_s do return to_s_unsafe
 
-	redef fun to_s_full(byte_length, unilen) do
-		return new FlatString.full(self, byte_length, 0, unilen)
-	end
+	# Get a `String` from `byte_length` bytes at `self` copied into Nit memory
+	#
+	# The string is cleaned.
+	fun to_s_with_length(byte_length: Int): String do return to_s_unsafe(byte_length)
```

Comment:
Same here, especially since the documentation states that the string is cleaned before being returned as a `String`